extensions/extension.py.example

```diff
@@ -0,0 +1,34 @@
+"""This file demos a few features of discord.py
+    It also provides a quick framework to make starting
+    a new extension quickly."""
+
+from discord.ext import commands
+
+# Name of the command that activates the game.
+# Currently, the bot will respond to the message '.test'
+COMMAND = 'test'
+
+class example(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name=COMMAND)
+    async def example_command(self, ctx):
+        """Does something."""
+        await ctx.send("This is an example.")
+
+    @commands.Cog.listener()
+    async def on_message(self, ctx):
```

Comment:
2nd argument of `on_message` is actually a Message object not a Context object so the variable should be `ctx`. The rest of the code looks good.